app/controllers/apps/letsencrypt_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Apps::LetsencryptController < ServerBaseController
+  def create
+    @app = server.apps.find(params[:app_id])
+    @app.fetch_letsencrypt_status_from_server!
+    unless @app.letsencrypt_enabled?
+      @app.assign_attributes(letsencrypt_params)
+      EnableLetsencryptJob.perform_later(@app) if @app.save
+    end
+
+    redirect_to [@app.server, @app, :domains]
```

Comment:
My preference would be to be explicit with where you route to, this requires extra brain cycles to determine what route this is (i.e. I can;'t tell you within 2 seconds what route this will create)